src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      crossorigin="anonymous"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <title>Javascript Group Capstone Project By David Yannick and Saningo  </title>
+</head>
+<body>
+  <header>
+    <div class="navbar">
+      <a class="link-logo" href="./index.html"> <img class="logo" src="../src/img/logo.png" alt="logo"></a>
+      <nav class="links">
+        <ul class="link-lists">
+          <li class="link-list"><a href="./index.html" class="link tv-show-link">.</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main class="main-content">
+    <div class="grid-cards-container"></div>
+  </main>
+  <aside class="movie-popup"></aside>
+  <footer>
+    Javascript Group Capstone By David Yannick and Saningo
+  </footer>
```

Comment:
- [ ] Kindly align  the `Home page footer`'s content to the left to make it similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). 

Your layout | Mockup layout
------------- | -------------
<img width="689" alt="Screenshot 2022-04-22 at 14 03 42" src="https://user-images.githubusercontent.com/77038610/164685020-d294ad5c-ddef-4cbf-b478-3c4639c74151.png">  | <img width="539" alt="Screenshot 2022-04-22 at 14 03 23" src="https://user-images.githubusercontent.com/77038610/164684980-ff495375-c2e2-4c5a-8e6f-691289a8e6fd.png">